### DISCUSSION TITLE
kernel is getting built but cant be stored onto the storage drive  ERROR!

### DISCUSSION THREAD

**Discussion by SwapnilVicky** [ROM-builders/temporary#15813] [Q&A]
Someone told me that below is the solution of the error that I'm getting in all my build, its started after the server had the storage full issue, please check sir
* Seems like kernel is getting built but cant be stored onto the storage drive
* See if the drive has full rw permissions 

Build 1 error: https://cirrus-ci.com/task/5998084031774720?logs=build#L8680
Build 2 error: https://cirrus-ci.com/task/5258948815618048?logs=build#L11142
Build 3 error: https://cirrus-ci.com/task/6375240276639744?logs=build#L7658

**Comment by Apon77** [comment]
I have to idea what to do to fix

**Reply by SwapnilVicky** [reply]
thanks
